terraform/main.mk

```diff
@@ -4,7 +4,7 @@ SSH_PUBLIC_KEY ?= $(shell cat ~/.ssh/id_rsa.pub)
 EC2_KEY_PAIR_NAME ?= $(ENV)-$(NAMESPACE)
 ENV_DIR ?= $(INFRA_DIR)/env/$(ENV)
 OUTPUT_JSON_FILE = $(INFRA_DIR)/env/$(ENV)/output.json
-TERRAFORM_VERSION ?= "0.12.29"
+TERRAFORM_VERSION ?= "0.13.3"
```

Comment:
Let's not upgrade the terraform version here, but instead mention setting `TERRAFORM_VERSION ?= "0.13.3"` in the `Makefile` in the upgrade documentation.

Reply:
done